hwloc/topology-rsmi.c

```diff
@@ -112,6 +122,59 @@ static int get_device_serial_number(uint32_t dv_ind, char *serial, unsigned int
   rsmi_status_t rsmi_rc = rsmi_dev_serial_number_get(dv_ind, serial, size);
 
   if (rsmi_rc != RSMI_STATUS_SUCCESS) {
+    if (!hwloc_hide_errors()) {
+      const char *status_string;
+      rsmi_rc = rsmi_status_string(rsmi_rc, &status_string);
+      fprintf(stderr, "RSMI: GPU(%u): Failed to get serial number: %s\n", (unsigned)dv_ind, status_string);
+    }
+    return -1;
+  }
+  return 0;
+}
+
```

Comment:
Can you split the 3 above hunks to a separate commit sine it's unrelated to XGMI?

Reply:
OK